web/client/components/widgets/builder/wizard/ChartWizard.jsx

```diff
@@ -118,13 +122,14 @@ module.exports = enhanceWizard(({ onChange = () => { }, onFinish = () => { }, se
             data={data}
             onChange={onChange}
             layer={data.layer || layer}
-            sampleChart={renderPreview({ data, layer: data.layer || layer, dependencies, setValid: v => setValid(v && isChartOptionsValid(data.options)), shortenChartLabelThreshold })}
+            sampleChart={renderPreview({ data, layer: data.layer || layer, dependencies, setValid: v => setValid(v && isChartOptionsValid(data.options)) })}
         />
         <WidgetOptions
             key="widget-options"
             data={data}
             onChange={onChange}
             layer={data.layer || layer}
-            sampleChart={renderPreview({ data, layer: data.layer || layer, dependencies, setValid: v => setValid(v && isChartOptionsValid(data.options)), shortenChartLabelThreshold })}
+            sampleChart={renderPreview({ data, layer: data.layer || layer, dependencies, setValid: v => setValid(v && isChartOptionsValid(data.options)) })}
         />
-    </Wizard>));
+    </Wizard>);
+export default enhanceWizard(ChartWizard);
```

Comment:
this is missing the display name, maybe is better to export a function called ChartWizard which contains the enhancer...
so it is easier to debug